src/material/schematics/ng-add/setup-project.ts

```diff
@@ -33,13 +33,21 @@ const noopAnimationsModuleName = 'NoopAnimationsModule';
  *  - Adds Browser Animation to app.module
  */
 export default function(options: Schema): Rule {
-  return chain([
-    addAnimationsModule(options),
-    addThemeToAppStyles(options),
-    addFontsToIndex(options),
-    addMaterialAppStyles(options),
-    addTypographyClass(options),
-  ]);
+  return (host: Tree) => {
+    const workspace = getWorkspace(host);
+    const project = getProjectFromWorkspace(workspace, options.project);
+
+    if (project.projectType === 'application') {
```

Comment:
Should we print a warning instead of exiting silently? Also would you be able to create a test for this?

Reply:
yeah, it's a good idea..
it would be great if we could prevent asking for ng-add parameters but as far as i know, x-prompt variables can't be defined conditionally based on project type, right?
sure, i'll provide a unit test case

Reply:
Yeah, I think we can't conditionally show prompts.